src/amazestats/routes.clj

```diff
@@ -71,7 +72,9 @@
            (GET "/:competition/divisions" [competition]
                 (find-divisions-by-competition competition))
            (GET "/:competition/teams" [competition]
-                (find-teams-by-competition competition)))
+                (find-teams-by-competition competition))
+           (GET "/:competition/admins" [competition]
+                (get-competition-admins competition)))
```

Comment:
Why is it get-competition-admins and not find-admins-by-competition? (Just out of curiosity)

Reply:
Good question, I thought about this as well, but did not take action to change it because I did not really have that convincing arguments for/against it at the time.

Reply:
Okay, then let me rephrase: Why is it find-teams-by-competition and not get-competition-teams? Is it only that it sound better or is there a reason for it? To me it looks like they are doing the exactly same thing?

Reply:
Honestly, I think I would prefix them all with `get` today. The reason (I think) behind the names are that I dabbled with allowing different query parameters (division, competition) to filter the result of a request for e.g. teams. Then we are asking the API to find teams based on some search criterion, whereas with a get I would expect the function call to be a bit more straight-forward.

Does that make sense? :D

Reply:
Yes, that makes sense. Thank you :)